CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+##  [1.0.0-rc.0] - Unreleased
```

Comment:
This is not meant to be merged as part of 1.0.0, it should be moved to Unreleased.